core/templates/dev/head/pages/exploration_editor/editor_tab/test_interaction_modal_directive.html

```diff
@@ -1,14 +1,19 @@
 <style>
-.submit-answer-btn {
-  text-align: right;
-}
+  .test-interaction-modal .submit-answer-btn {
+    text-align: right;
+  }
+
+  .test-interaction-modal .enter-answer-label {
+    font-weight: bold;
+    text-indent: 8%;
+    margin-bottom: 5%;
+  }
 </style>
 
-<div class="form-inline" style="margin-bottom: 20px;">
+<div class="form-inline test-interaction-modal" style="margin-bottom: 20px;">
   <div>
-    <h3> Enter a sample answer </h3>
+    <p class='enter-answer-label'> Enter a sample answer </p>
```

Comment:
Double quotes please. /cc @apb7 @kevinlee12

Reply:
Added in Issue #5108. Thanks!

Reply:
Done. Sorry, I am habituated to single quotes.